test/test_distributions.py

```diff
@@ -1603,6 +1603,58 @@ def ref_log_prob(idx, x, log_prob):
 
         self._check_log_prob(Normal(loc, scale), ref_log_prob)
 
+    def test_normal_shape_cpu(self):
+        # CPU version normal is written in legacy code (TH).
+        # It won't support tensor broadcasting until it is migrated using new tensor iterator.
+        tensor23 = torch.rand(2, 3)
+        tensor21 = torch.rand(2, 1)
+        tensor6 = torch.rand(6)
+        tensor4 = torch.rand(4)
+        # normal case
```

Comment:
Probably better to name this something else, like "inputs have same size" to avoid confusion with the  op name